dodder-common/src/test/java/cc/dodder/common/util/StringUtilTest.java

```diff
@@ -0,0 +1,80 @@
+package cc.dodder.common.util;
+
+import cc.dodder.common.entity.Node;
+import cc.dodder.common.entity.Torrent;
+import org.junit.Rule;
+import org.junit.rules.ExpectedException;
+import org.junit.Test;
```

Comment:
Usually, similar imports are sorted by uppercase, then lowercase so `import org.junit.Test;` should come before `import org.junit.rules.ExpectedException;`